Solution4.py

```diff
@@ -0,0 +1,28 @@
+# Question 4 Solution
+import csv
+header=['sid','name','salary']
+rows=[]; L=[]
+temp=1     #counter
```

Comment:
WHY WE INITIALISED TEMP VARIABLE WITH 1 ?

Reply:
there is no need for it to be one, it can be any number which is not 0